.gitignore

```diff
@@ -29,3 +29,10 @@ package-lock.json
 **/__snapshots__/**
 src/components/dashboard/container/ListContainer.jsx
 .env
+
+tags
+tags.lock
+tags.temp
+
+*.swp
+*.swo
```

Comment:
Is this related to the ticket?

Reply:
no. Those are vim generated files so I didn't want them to end up on the repo.

Reply:
Hey @muhozi, I have fixed it. I created a global `.gitignore` file for my editor's generated files.